### PR TITLE
add radseq post

### PR DESCRIPTION
**Pull request by bgruening**
@yvanlebras is that ok for you?

### CODE REVIEW

**Comment by yvanlebras** [comment]
Perfect! Thank you!! (And more tweets to come this weekend ;) #hopebabywillsleepalot )

File: _posts/2021-03-13-radseq-analysis.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+site: [freiburg]
+title: RADseq data analysis (on mobile phone while baby is sleeping)
+tags:
+  - data
+  - tools
+  - elixi
```
Comment (beatrizserrano) [inline _posts/2021-03-13-radseq-analysis.md:7]:
```suggestion
  - elixir
```

File: _posts/2021-03-13-radseq-analysis.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+[Yvan Le Bras](https://twitter.com/Yvan2935) reminded us today that everyone can analyse RADseq data in Galaxy.
+
+[RADSeq](https://doi.org/10.1093/bfgp/elq031) combines two simple molecular biology techniques with
+Illumina sequencing: the use of restriction enzymes to cut DNA into fragments (as for RFLPs and AFLPs),
+and the use of molecular identifiers (MID) to associate sequence reads to particular individuals.
+This delivers high resolution population genomic data — thousands of sequenced markers across many individuals — for any organism at reasonable costs.
```
Comment (beatrizserrano) [inline _posts/2021-03-13-radseq-analysis.md:15]:
```suggestion
This delivers high-resolution population genomic data — thousands of sequenced markers across many individuals — for any organism at reasonable costs.
```

**Review by beatrizserrano — APPROVED** [review]

File: _posts/2021-03-13-radseq-analysis.md (hunk 1 above)
Comment (beatrizserrano) [reply _posts/2021-03-13-radseq-analysis.md:7]: not sure now about the elixir tag, we don't have supporters as tags...

**Pull request merged**